vector/build.gradle

```diff
@@ -131,6 +136,9 @@ android {
         // Required for sonar analysis
         versionName "${versionMajor}.${versionMinor}.${versionPatch}-sonar"
 
+        // Generate a random app task affinity
+        manifestPlaceholders = [appTaskAffinitySuffix:"${generateTaskAffinitySuffix()}"]
```

Comment:
Ah, something else I did not think about.
We want to have stable builds, i.e. the same code generates the same binary (related: #842, and context in https://github.com/vector-im/riot-android/issues/3131). It's important for F-Droid.
With this change it will not be the case anymore.
I have not idea how to handle that now.

Reply:
could we use the a short git hash for the suffix? :thinking:

Reply:
We should propably compute the task affinity suffix from tags or commit hash ?

Reply:
> could we use the short git hash for the suffix? 🤔

We can't use it as is. taskAffinity has same format namespace : 
- It must have at least two segments (one or more dots).
- Each segment must start with a letter.
- All characters must be alphanumeric or an underscore [a-zA-Z0-9_].

https://developer.android.com/studio/build/configure-app-module#set-application-id

Reply:
we could use the git hash as a seed for the `random()`

Reply:
That's a good id to compute a suffix using git hash and just remove forbidden chars (but is there any?).

Reply:
So many having a `appTaskAffinitySuffix` like `a${git_hash_8_digits}` with a `a` prefix to ensure it's always starting with a letter would fit all our needs. WDYT @yostyle ?

Reply:
we already have `gitRevision`. I'm going to reuse it.